### PR TITLE
Release

### PR DESCRIPTION
**Pull request by adebsa2401**
In this PR,

- we built the homepage
- we built the details page
- we set unit tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,185 @@
-# Getting Started with Create React App
+<a name="readme-top"></a>
```
Comment (TedLivist) [inline README.md:2]: - [ ] Kindly record a video presentation of your project and include the link in your README.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,185 @@
...
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+- Microverse
+
```
Comment (TedLivist) [inline README.md]: - [ ] Kindly add the Acknowledgment section and include credits to Nelson Sakwa whose design was used for this project.

File: src/pages/HomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+              flag={country.flag}
+              map={country.map}
+              iso2={country.iso2}
+            />
+          ))}
+        </div>
+      </main>
+    </div>
+  );
+}
```
Comment (TedLivist) [inline src/pages/HomePage.jsx]: - [ ] On your homepage, kindly include the intro banner as we have in the design, with more description of the page content.

  For example, in the design, we have a banner section with "Europe" on it, indicating that the countries on the page are European countries with their views data (count).

  akumu-descriptive6

File: src/pages/HomePage.jsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Filter from '../components/Filter';
+import Header from '../components/Header';
+import CountryItem from '../components/CountryItem';
+import { loadCountries } from '../redux/countries/countries';
+import '../styles/HomePage.css';
```
Comment (TedLivist) [inline src/pages/HomePage.jsx:7]: - [ ] For all your pages, kindly ensure you maintain the font style as we have on the design template.

  Your font style | Design font style
  :---:|:---:
  dkkd | idoee

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @adebsa2401,

Good job so far!
✔️ Implementation of alternating colors
✔️ Implementation of filter on the homepage
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @adebsa2401, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**